templates/component-rust/metadata/spin-template.toml

```
@@ -0,0 +1,12 @@
+manifest_version = "1"
+id = "component-rust"
+description = "Pure Rust component to be used as a dependency for other component"
```

Remove "pure" e.g.

```suggestion
description = "Library (dependency) component using Rust"
```

or something like that?